.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: kacl-verify
+  name: "verify kacl/changelog"
+  description: 'Veryfies if the changelog is in "keep-a-changelog" format.'
+  entry: kacl-cli verify
+  language: python
+  types: [python]
```

Comment:
I think we only need to verify `CHANGELOG.md`, so instead of `types: [python]`, I think it should be `files: '^CHANGELOG.md`

Reply:
@alfechner this is great, thanks for the PR. Could you please change the hook according to @Qu4tro request?